src/_data/languages_base.json

```diff
@@ -71,6 +73,7 @@
     "rssFeedHeading": "RSSフィード",
     "rssFeedParagraph": "OWAの最新ニュースを購読する。",
     "siteFooterPoweredBy": "このサイトはNetlifyを利用しています",
-    "skipLinkText": "コンテンツにスキップする"
+    "skipLinkText": "コンテンツにスキップする",
+    "translationNotAvailable": "残念ながら、このページはまだ日本語に翻訳されていません。翻訳にご協力いただける場合は、OWA Discord の #translation にご参加ください。"
```

Comment:
Suggestion: we could link to the Discord invite URL here so that visitors don't need to find Discord link from other pages:

![Screenshot of deploy preview link. text "OWA Discord の #translation" is selected](https://github.com/user-attachments/assets/41700fd9-c050-4366-b35e-1471f8d7181b)

You can implement it with the following change:

```suggestion
    "translationNotAvailable": "残念ながら、このページはまだ日本語に翻訳されていません。翻訳にご協力いただける場合は、<a href=\"https://discord.gg/x53hkqrRKx\">OWA Discord の #translation</a> にご参加ください。"
```

and then add `safe` filter in `banner-translation.njk` (to avoid HTML escaping):

```diff
-  <p>{{ content.translationNotAvailable }}</p>
+  <p>{{ content.translationNotAvailable | safe}}</p>
```

Reply:
Oh that's cool, I did not know about the `safe` filter! Thank you.